### PR TITLE
taipy 3.0 (#82)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
- Sequence management
- Configuration file upgrade

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,27 +53,27 @@ selected configuration file.
...
-  - If the Task is already present in the Pipeline or Scenario
+  - If the Task is already present in the Sequence or Scenario
     diagram view that is opened, the view will be panned so
     that this Task is displayed in its center.
-  - If the Task is not present in the Pipeline or Scenario
+  - If the Task is not present in the Sequence or Scenario
     diagram view that is opened, it is added to it.
 
-### Pipelines section
+### Sequences section
```
Comment (FabienLelaquais) [inline README.md]: What is that?

File: package.json [hunk 2]
```diff
@@ -245,29 +258,38 @@
...
+        {
+          "command": "taipy.scenario.showSequence",
+          "when": "taipy.config.diagram.visible && viewItem == SEQUENCE && view == taipy-config-scenarios"
         }
       ],
       "webview/context": [
         {
           "command": "taipy.perspective.showFromDiagram",
-          "when": "webviewId == 'taipy.config.editor.diagram' && webviewSection =~ /taipy.*-persp$/"
+          "when": "webviewId == 'taipy.config.editor.diagram' && webviewSection =~ /taipy.*-persp/"
```
Comment (FabienLelaquais) [inline package.json:292]: Did that ever worked??

File: schemas/config.schema.json [hunk 3]
```diff
@@ -13,13 +75,12 @@
       "additionalProperties": {
         "properties": {
           "scope": {
-            "description": "An enum value among GLOBAL, CYCLE, SCENARIO or PIPELINE that represents the visibility of the data node in the graph of entities.",
+            "description": "An enum value among GLOBAL, CYCLE, SCENARIO that represents the visibility of the data node in the graph of entities.",
```
Comment (FabienLelaquais) [inline schemas/config.schema.json:78]: "CYCLE, SCENARIO" -> "CYCLE, or SCENARIO"

**Review by FabienLelaquais — APPROVED** [review]
🤨

**Review by dinhlongviolin1 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (FredLL-Avaiga) [reply README.md]: reminiscence of change all, I'll fix that

File: package.json (hunk 2 above)
Comment (FredLL-Avaiga) [reply package.json:292]: yes it did because -persp was allow last :-)

File: schemas/config.schema.json (hunk 3 above)
Comment (FredLL-Avaiga) [reply schemas/config.schema.json:78]: I am not responsible for this file: it is a straight copy of https://github.com/Avaiga/taipy-core/blob/develop/src/taipy/core/config/config.schema.json

Comment (FredLL-Avaiga) [reply schemas/config.schema.json:78]: @jrobinAV Can you take a look on your side?
I have also seen quite a few description that are not adequate

**Pull request merged**